apps/spruce/src/pages/spawn/spawnHost/SpawnHostTableActions.test.tsx

```diff
@@ -203,6 +203,41 @@ describe("spawn host table", () => {
       screen.queryByDataCy("pause-unexpirable-host-button"),
     ).not.toBeInTheDocument();
   });
+
+  it("does not prompt user when permanent exemption is granted", () => {
+    const { Component } = RenderFakeToastContext(
+      <SpawnHostTable
+        hosts={[
+          {
+            ...baseSpawnHost,
+            sleepSchedule: {
+              ...defaultSleepSchedule,
+              nextStartTime: null,
+              timeZone: "America/New_York",
+              permanentlyExempt: true,
+            },
+          },
+        ]}
+      />,
+    );
+    render(
+      <MockedProvider
+        mocks={[
+          getSpruceConfigMock,
+          getUserSettingsMock,
+          instanceTypesMock,
+          myHostsMock,
+          myPublicKeysMock,
+          myVolumesQueryMock,
+        ]}
+      >
+        <Component />
+      </MockedProvider>,
+    );
+    expect(
+      screen.queryByDataCy("pause-unexpirable-host-button"),
+    ).not.toBeInTheDocument();
+  });
```

Comment:
would you be opposed to fixing the `// ts-expect-error` for the `copySSHButton` statements in this file 👉👈 it should be very easy, just replace `queryByDataCy` with `getByDataCy`!